corehq/apps/domain/models.py

```diff
@@ -1227,3 +1229,46 @@ def as_dict(self):
             'deactivate_url': self.deactivate_url(),
             'activate_url': self.activate_url(),
         }
+
+
+class DomainAuditRecordEntry(models.Model):
+    id = models.UUIDField(unique=True, default=uuid.uuid4, primary_key=True)
+    domain = models.TextField(unique=True, db_index=True)
+    calculations = JSONField(default=dict)
```

Comment:
What about adding a column field for each calculated value instead of having a JSON field?

Reply:
:+1: now I see why column for each calculated value will be better. We don't need to worries to update old records in the case when we add new calculated properties.